swarm/src/main/resources/swarm/swarm.bom

```diff
@@ -53,7 +53,7 @@ brooklyn.catalog:
           description: |
             Size of the etcd cluster when created initially
           type: integer
-          default: 1
+          default: 2
```

Comment:
Doesn't etcd need an odd number of cluster members? Does it just ignore one of the members is they are even?